src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/ArgumentAnalysisResult.cs

```diff
@@ -86,8 +86,10 @@ public static ArgumentAnalysisResult ExpandedForm(ImmutableArray<int> argsToPara
             return new ArgumentAnalysisResult(ArgumentAnalysisResultKind.Expanded, 0, 0, argsToParamsOpt);
         }
 
-#if DEBUG        
+#if DEBUG
+#pragma warning disable IDE0051 // Remove unused private members
```

Comment:
Consider adding an Obsolete attribute and making the "unused member" analysis recognize the Obsolete attribute (ie. shouldn't complain).

Reply:
I'm not sure why this method would be obsolete? It can be used for debugging purposes right?

Reply:
Sure, but we don't reference it from source. Typically we invoke it from EE.
Fine to leave as-is

Reply:
If that's the case, pls consider adding a comment saying that this is invoked dynamically.